### PR TITLE
install cypress/allure dependencies

### PR DESCRIPTION
**Pull request by TyStelmach**
### Summary of Changes 📋
We moved the defaults into this library in order to make setting up allure visualizations easier. We're running into some missing dependency issues, so we're also going to include them with this library. 

Blog PR: https://git.hubteam.com/MarketingWebTeam/blog/pull/973
Bug Ticket: https://issues.hubspotcentral.com/browse/FDNDEVOPS-2858

### CODE REVIEW

**Comment by swatinigam** [comment]
👍

File: cypress.config.js [hunk 1]
```diff
@@ -42,34 +42,40 @@ const getDevBaseUrl = () => {
...
 };
 
-const e2e = {
-  specPattern: 'cypress/e2e/*.cy.js',
-  testIsolation: false,
-  async setupNodeEvents(on, config) {
-    const webpackOptions = {
-      resolve: {
-        extensions: ['.ts', '.js'],
+async function setupNodeEvents(on, config) {
```
Comment (TyStelmach) [inline cypress.config.js:45]: I tried to handle this all in a single object, but when imported, the function is not working as expected, so this must be standalone.

**Review by darkmavis1980 — APPROVED** [review]

**Review by darkmavis1980 — CHANGES_REQUESTED** [review]
Actually, the workflows are failing, have a look at the logs

**Comment by TyStelmach** [comment]
> Actually, the workflows are failing, have a look at the logs

This should be passing now, let me know if you see anything else on your end. @darkmavis1980

**Review by darkmavis1980 — APPROVED** [review]

**Review by cfrisoli — APPROVED** [review]

**Pull request merged**